contracts/test/utils.ts

```diff
@@ -1,38 +1,58 @@
+import { constants, ec, encode, hash, number, uint256, stark, KeyPair } from 'starknet'
```

Comment:
Don't need all of it we can change it to `import { constants, encode, number} from 'starknet'`